portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,24 +25,43 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<String>();
-    comments.add("Hello");
-    comments.add("How are you?");
-    comments.add("Greetings");
-
+    System.out.println("I do GET");
     //Convert comments arraylist to JSON
     String json = convertToJson(comments);
 
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    System.out.println("I do POST");
```

Comment:
Avoid keep System.out.println on the submitted code. They are a great help during development, but in the final code that avoid the code cleanness and readability.

Reply:
Will not happen again!